bot/controllers/channels.js

```diff
@@ -3,7 +3,7 @@ const Command = require('../baseCommand.js');
 const util = require('apex-util');
 const msg = require('../locale/messages.json');
 
-const LANGUAGE = process.env.LANGUAGE;
+const lang = process.env.LANGUAGE;
```

Comment:
Make a provision for setting the default language to en, incase no language is set in the .env.